docs/guides/Glossary.md

```diff
@@ -1,13 +1,19 @@
-# Glossary
+## Glossary
 
-## Implicit cast
+### Others bucket
```

Comment:
Shoud we order these entries (move 'Others' before 'Variables')?

Reply:
:+1: